cou/apps/auxiliary.py

```diff
@@ -183,6 +183,92 @@ class RabbitMQServer(AuxiliaryApplication):
     # COU changes to 3.9 if the channel is set to 3.8
     multiple_channels = True
 
+    def upgrade_plan_sanity_checks(
```

Comment:
I do not think that we should overwrite this function, but rather the `_check_auto_restarts`.

Reply:
Done